src/app/app-home/app-home.component.scss

```diff
@@ -38,3 +38,7 @@ mat-sidenav-container {
 .content {
   overflow: scroll;
 }
+
+.disabled {
+  color: darkgray !important;
```

Comment:
I guess we need important to override angular material?

Reply:
exactly